src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -538,7 +542,7 @@ private static async IAsyncEnumerable<Regression> FindRegression(Source source)
                         var result = new BenchmarksResult
                         {
                             Id = Convert.ToInt32(reader["Id"]),
-                            Excluded = Convert.ToBoolean(reader["Excluded"]),
+                            Excluded = Convert.ToBoolean(reader["Excluded"] as bool? ?? false), // Handle DBNull values
```

Comment:
nit: the `Convert` call doesn't look like it's needed

Reply:
Fixed